airbyte-workers/build.gradle

```diff
@@ -143,6 +143,7 @@ task generateWellKnownTypes() {
 
 tasks.named("buildDockerImage") {
     dependsOn copyGeneratedTar
+    dependsOn generateWellKnownTypes
```

Comment:
not sure I understand this - buildDockerImage already depends on copyGeneratedTar, which depends on generateWellKnownTypes. So isn't this dependency redundant?

Reply:
Are we copying this file in the tar?

Reply:
I suspect the monorepo project is going to impact those dependency, I think it might make sense to punt on dependency clean up for now.

Reply:
welp. sounds good, lgtm